### PR TITLE
Enrol MOJ root account into central SecurityHub

### PR DESCRIPTION
**Pull request by jakemulley**
This PR sets the precedence for enrolling accounts into central SecurityHub, and is the same as how [guardduty.tf](https://github.com/ministryofjustice/aws-root-account/blob/main/terraform/guardduty.tf) works.

### CODE REVIEW

File: terraform/securityhub.tf [hunk 1]
```diff
@@ -24,3 +30,38 @@ resource "aws_securityhub_organization_admin_account" "default-region-administra
...
+  # See: https://docs.aws.amazon.com/guardduty/latest/APIReference/API_UnprocessedAccount.html
+  for_each = {
+    for account in local.enrolled_into_securityhub :
+    account.name => account.id
+  }
+  provider = aws.organisation-security-eu-west-2
+
+  # We want to add these accounts as members within the Organisation Security account
+  account_id = each.value
+  email      = "fake@email.com"
```
Comment (SteveMarshall) [inline terraform/securityhub.tf:50]: `email.com` is a real live domain. For this sort of thing, you should always use an IANA-

Comment (SteveMarshall) [reply terraform/securityhub.tf:50]: (Unless AWS blocks that, in which case, let's reserve `example.justice.gov.uk`.)

**Review by SteveMarshall — CHANGES_REQUESTED** [review]

File: terraform/securityhub.tf (hunk 1 above)
Comment (jakemulley) [reply terraform/securityhub.tf:50]: Ooo, thanks @SteveMarshall 👍🏼

**Review by SteveMarshall — APPROVED** [review]
:shipit:

**Pull request merged**